Rocket.Chat/Resources/cs.lproj/Localizable.strings

```diff
@@ -366,7 +366,7 @@
 "myaccount.settings.notifications.duration.default" = "Default"; // TODO
 "myaccount.settings.notifications.duration.seconds" = "seconds"; // TODO
 
-"alert.update_notifications_preferences_success.title" = "Successfully updated your notification preferences."; // TODO
+"alert.update_notifications_preferences_success.title" = "Perferences saved"; // TODO
```

Comment:
Typo :) "P**re**ferences saved".

Reply:
Sometimes I hate that Xcode doesn't have autocorrect for strings. 🙈